setup.cfg

```diff
@@ -182,6 +180,10 @@ filterwarnings =
     ignore:FLIP_TOP_BOTTOM is deprecated and will be removed in Pillow 10:DeprecationWarning
     # Deprecation warning added in ImageIO 2.16.2, called by scikit-image
     ignore:Starting with ImageIO v3 the behavior of this function will switch:DeprecationWarning
+    ignore:`multichannel` is a deprecated argument name for `rescale`:FutureWarning
+    ignore:divide by zero encountered in double_scalars
+    ignore:All-NaN slice encountered
+    ignore:ASDF functionality for astropy is being moved out of the astropy package
```

Comment:
Can we try and add these to individual tests instead of globally? That way it makes it easy to see that they're expected warnings in context.